### ISSUE TITLE
Disable coloring of blocks with right-click of dye

### ISSUE THREAD

**Issue by jvgoch** [refinedmods/refinedstorage#2877]
#### Issue description:
When I right click a Refined Storage block it changes color instead of opening the GUI. This is extremely annoying while trying to craft with dyes.... **(I know this is intended behavior)**
I would really like to have this changed to a shift + right-click or have it be disabled all together. You would still have the ability to change the color in a crafting grid. I suppose you could also have a button on the left side of the GUI's that you could click with a dye, this would be a different way to change it without destroying the block.
#### What happens:
The block changes color and I lose my dye.
#### What you expected to happen:
I would like the GUI to open.
#### Steps to reproduce:
1. Right click block
2. Lose Dye
3. Be annoyed
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.16.5
- Forge: 36.0.14
- Refined Storage: 1.9.11

**Comment by AWDoBaun** [comment]
I agree completely and would also like to point out that I very much did not want a coloured grid and that there is no way to re-craft that grid into the uncoloured version.

**Comment by Darkere** [comment]
They should be craftable/dyeable with lightblue dye to get back to the original color.

**Comment by AWDoBaun** [comment]
The grid dyed Cyan looks very similar, but not the same as the original but it is not the original block, it is "Cyan Crafting Grid" not "Crafting Grid".

**Comment by Darkere** [comment]
Thats why I said light blue and not cyan :)

**Comment by raoulvdberge** [comment]
Hi! Thanks for the suggestion. This has been added to the RS 2 tracker, and will be implemented for RS 2.x. 

Closing this issue for RS 1.x on this repository.
